src/CoreShop/Bundle/VariantBundle/Twig/Extension/VariantExtension.php

```diff
@@ -76,7 +76,7 @@ public function serializeGroups(array $attributeGroups, array $groups = ['coresh
         return $this->serializer->normalize($attributeGroups, 'json', [
             'groups' => $groups,
             AbstractNormalizer::CALLBACKS => [
-                'valueColor' => static function (?string $innerObject) {
+                'valueColor' => static function (string $innerObject) {
```

Comment:
Was this change intentional? Because it was introduced recently in #2058.

Reply:
many thanks @jdreesen, this was by mistake. here is the PR with the revert: https://github.com/coreshop/CoreShop/pull/2069